src/modules/kvs/kvs.c

```diff
@@ -2387,6 +2401,23 @@ static void stats_get_cb (flux_t *h,
                               "#faults", ctx->faults)))
         goto nomem;
 
+    if (!(txncstats = json_pack ("{ s:i s:i s:f s:f s:i }",
+                                 "count", tstat_count (&ctx->txn_commit_stats),
+                                 "min", (int)tstat_min (&ctx->txn_commit_stats),
+                                 "mean", tstat_mean (&ctx->txn_commit_stats),
+                                 "stddev", tstat_stddev (&ctx->txn_commit_stats),
+                                 "max",
+                                   (int)tstat_max (&ctx->txn_commit_stats))))
+        goto nomem;
+
+    if (!(txnfstats = json_pack ("{ s:i s:i s:f s:f s:i }",
+                                 "count", tstat_count (&ctx->txn_fence_stats),
+                                 "min", (int)tstat_min (&ctx->txn_fence_stats),
+                                 "mean", tstat_mean (&ctx->txn_fence_stats),
+                                 "stddev", tstat_stddev (&ctx->txn_fence_stats),
+                                 "max", (int)tstat_max (&ctx->txn_fence_stats))))
+        goto nomem;
```

Comment:
This is the same as the `tstats` object except `max` is packed as an intetger.

Would adding a `add_tstat_to_object()` function be useful here to get all three in the same format, reduce code duplication, and avoid the need for a temporary variable per object?

Reply:
Sounds like a good idea.  However, I do dislike that the min and max for op-count would be a double if we created a common function for this.  Although it's not the end of the world if that were the case.  

For the cache stats, I suppose min and max should also always be integers (or big Ints) too.  However, that would be changing the output format.  Hmmm, lemme ponder this detail for a bit.

Reply:
Changing the output format here doesn't seem so bad.